sklearn/utils/validation.py

```diff
@@ -1076,7 +1076,11 @@ def check_X_y(
         performed if the dtype of the input is not in the list.
 
     order : {'F', 'C'}, default=None
```

Comment:
@betatim , you're completely right! On the first search not sure how I got to the `mocking` file instead of `validation`. Nonetheless, I've now addressed the actual missing `None` in `check_X_y`.